code/repo_sync

```diff
@@ -819,7 +819,7 @@ def main():
                                         reposadocommon.pref('CurlPath'))
             exit(-1)
         if (not reposadocommon.pref('LocalCatalogURLBase') or
-                reposadocommon.pref('SkipPackages')):
+                reposadocommon.pref('AlwaysRewriteDistributionURLs')):
             download_packages = False
```

Comment:
Now that I look at this some more I see that is is likely to lead to confusion. I think we're going to need better documentation at least. It might not be clear/obvious that in order to _not_ replicate pkgs, but to be able to modify and serve distribution files, that you need to define `LocalCatalogURLBase` and set `AlwaysRewriteDistributionURLs`.  It might not be clear/obvious that setting `AlwaysRewriteDistributionURLs` causes pkgs to _not_ be replicated. Your original two keys made that a bit clearer, but implied at least six possible states (and six combinations to test); one key implies fewer possible states and combinations to test and document.

So: any cleverer ideas about key naming to make this clearer?